repotracker/repotracker.go

```diff
@@ -710,6 +711,7 @@ func CreateVersionFromConfig(ctx context.Context, projectInfo *model.ProjectInfo
 	projectInfo.IntermediateProject.Id = v.Id
 	projectInfo.IntermediateProject.CreateTime = v.CreateTime
 	v.Ignored = ignore
+	v.Activated = utility.ToBoolPtr(false)
```

Comment:
Sorry! I didn't see this the first time. I'd recommend also doing utility.FalsePtr() here. 
This will cover all versions created for the waterfall. If this is something you also want for patches, you'll need to add it [here](https://github.com/evergreen-ci/evergreen/blob/0a647179c436aaa0a6c5e58c53c9524d6b1f1b04/model/patch_lifecycle.go#L309).

Reply:
Thanks i only need this for the waterfall but it wont hurt to add it!